laravel-directory-sync-app/app/Http/Controllers/GroupsController.php

```diff
@@ -52,7 +52,8 @@ public function show($id, $groupId)
     {
         $directoryGroupId = $groupId;  // ... The ID of the Directory Group to fetch
         $group = (new \WorkOS\DirectorySync())->getGroup($directoryGroupId);
-        return view('group', ['group' => $group]);
+        var_dump($group);
```

Comment:
Can you delete this var_dump? It's printing the json out on the groups page. 
![Uploading CleanShot 2023-03-31 at 13.38.49.png…]()